internal/cmd/commands/database/init.go

```diff
@@ -116,6 +118,12 @@ func (c *InitCommand) Flags() *base.FlagSets {
 
 	f = set.NewFlagSet("Init Options")
 
+	f.BoolVar(&base.BoolVar{
+		Name:   "allow-development-migration",
+		Target: &c.flagAllowDevMigration,
+		Usage:  "If set the init will continue even if the schema includes unsafe database update steps that have not been finalized.",
```

Comment:
It might be worth adding another sentence here indicating that there is no rollback mechanism, so a backup should be taken independently if needed.